stylesheets/scss/style.scss

```diff
@@ -15,9 +15,11 @@
 // Fonts, mixins, variables, etc
 @import 'includes/fonts';
 @import 'includes/mixins';
+@import 'includes/functions';
```

Comment:
@reubenmoes Hadn't really put thought on the order, but now that you mention it, it makes more sense to have functions on top of mixins.